spec/features/repair_request_spec.rb

```diff
@@ -298,15 +298,17 @@ def stub_property_temp_annex
       expect(page).to have_css(".hackney-property-warning-label-turquoise")
       click_on 'Raise a repair on this dwelling'
 
-      expect(page).to have_content "CC"
+      page.should have_css('.hackney-cautionary-contact-table')
```

Comment:
we should expect not should which is deprecated
https://github.com/rspec/rspec-expectations/blob/master/Should.md

Reply:
also, _if possible_, it would be nice to avoid asserting on invisible aspects of the page, such as CSS classes and ids.

Reply:
> also, _if possible_, it would be nice to avoid asserting on invisible aspects of the page, such as CSS classes and ids.

Problem here is that the page should have "CC" in two places, so I need to check that the table is there to check it's the right one.

Reply:
fair enough :sparkles: